### DISCUSSION TITLE
Bind a radio to array

### DISCUSSION THREAD

**Discussion by randohinn** [livewire/livewire#4502] [1. Help]
Need some really quick help - is there a way to get a wire:model that is mapped from a radio button, to an empty array to keep itself as an array, but with a single element?

Currently mine turns to a string of the selected radio's value... SO instead of having this (which I want) `[0]` I have `"0"`

**Comment by amiranagram** [comment]
I don't think this is possible. If you add an index to the `wire:model` you will be able to achieve that.

```html
<input wire:model="arr.0" type="radio" name="some_radio" value="0"/>
```
